pkg/booleanpolicy/augmentedobjs/construct.go

```diff
@@ -150,7 +151,10 @@ func ConstructDeployment(deployment *storage.Deployment, images []*storage.Image
 	}
 
 	for i, image := range images {
-		augmentedImg, err := ConstructImage(image)
+		// Since we ensure that both images and containers have the same length, this will not lead to index out of
+		// bounds panics.
+		containerImageFullName := deployment.GetContainers()[i].GetImage().GetName().GetFullName()
```

Comment:
is `deployment.GetContainers()[i].GetImage().GetName().GetFullName()` never empty?

Reply:
Yeah, `containers` have to specify an image name to be valid.